Makefile

```diff
@@ -191,6 +191,18 @@ test-cmd-devfile-list: install ## Run odo list devfile command tests
 test-cmd-devfile-create: install ## Run odo create devfile command tests
 	$(RUN_GINKGO) $(GINKGO_FLAGS) -focus="odo devfile create command tests" tests/integration/devfile/
 
+.PHONY: test-cmd-devfile-init
```

Comment:
what is this? Do we need to have this in makefile? Our make file is already too big, with a lot of  things that are rarely used. If someone has a need to run a specific test for only one command, they can always do it by executing `ginkgo` or `go test` we don't need to make a target for every comamnds

Reply:
ok, I can remove them. it is just painful to type huge commands every time especially without completion and I dont h\want to wait the time it takes to run everything :(
Guess il need to have my own make file :P